visualization/conf/webpack.loaders.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
 		},
 		{
 			test: /\.ts(x?)$/,
-			use: ["ng-annotate-loader", "babel-loader", "ts-loader?configFile=tsconfig.webpack.json"]
```

Comment:
The annoate-loader was there to support sourcemaps, if I am not mistaken. Are those still working as before?

Reply:
I just saw that this is outlined in the description. Resolved.